docs/cnd-file.md

```diff
@@ -40,3 +42,14 @@ The local folder synched to the remote container. (default: the current folder)
 ## mount.target (required)
 
 The remote folder path synched with the local file system.
+
+## swap.scripts (optional)
```

Comment:
is it inside `swap` or not? then, the example above is wrong.

Reply:
the example is wrong, I'll fix it. `scripts` should be a first level element